### ISSUE TITLE
Firm Overview 1: Justify Margins on Text

### ISSUE THREAD

**Issue by d4mation** [realbig/RBM_TH_INOSENCIO#16]
> We think that we should justify the margins on this text (this comment goes for
the entire website).

I need confirmation on whether they mean what they are saying. Justified text is different than centered text, and I'm thinking they may mean centered.

![image](https://user-images.githubusercontent.com/7770631/30979012-4793e1fc-a44a-11e7-8bf9-9fe536663143.png)

**Comment by BigActual** [comment]
Yes, justified. Sorry I didn't see this one when I created #21.

**Comment by d4mation** [comment]
Justified text is actually one of the very, very few things that is only supported in IE (and as of very recently, Firefox). 
http://caniuse.com/#search=text-justify

We can enable this and it will work where it can (And fallback to `text-align: left` where it isn't).

caniuse says it isn't supported in Chrome, but it does seem to work for me. It may not be a huge deal.

**Comment by d4mation** [comment]
I have applied Justified Text Alignment to all of the _main content areas_. It isn't quite "site-wide" because in some places (footer, headers, sidebar, etc.) I did not think it made much sense to have flat left and right edges to the text blocks.

If there are more sections that are desired to have this effect, I'll need to know which ones specifically. All _main content areas_ site-wide have Justified Text now.
